modules/daggerverse/main.go

```diff
@@ -21,6 +21,7 @@ type Daggerverse struct {
 	Repo string
 }
 
+// +cache="session"
```

Comment:
Is this one needed? 🤔

Reply:
Ah sorry didn't look close enough and assumed it was just a dumb constructor. LGTM 👍